build.gradle

```diff
@@ -5,7 +5,8 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '3.3.1'
+version '3.3.2'
+
```

Comment:
This is a breaking change thus it needs to be a MAJOR bump. Think Fergus is about to release 4.0.0 so maybe choose 5.0.0